trajan/traj.py

```diff
@@ -63,6 +64,39 @@ def __init__(self, ds, obsdim, timedim):
         self.obsdim = obsdim
         self.timedim = timedim
 
+    def __repr__(self):
+        output = '=======================\n'
+        output += 'TrajAn info:\n'
+        output += '------------\n'
+        output += f'{self.ds.sizes["trajectory"]} trajectories\n'
+        if self.timedim in self.ds.sizes:
+            output += f'{self.ds.sizes[self.timedim]} timesteps\n'
+        else:
+            logger.warning(f'self.timedim ({self.timedim}) is not an existing dimension!\n')
```

Comment:
In this case the number of time steps is 1?

Reply:
Would you call it a timestep if there is no associated time?

Reply:
Hehe good point, maybe observation. We still have one sample. It's a pity that time is lost when averaged.